src/actions/exercisesActions.js

```diff
@@ -95,3 +100,27 @@ export const deleteExercise = (id) => ({
     method: 'delete',
   },
 });
+
+export const patchExercise = (id, title, text, categoryID, batchID, fileUpload) => ({
```

Comment:
`ID` should be `Id`.
I would use `(id, exercise)` definition where `exercise` is an object.